docker/docker-compose.yml

```diff
@@ -97,6 +97,43 @@ services:
       <<: *fbstest-env
     <<: *ports
 
+  # fbstest with redis
+  fbstest-redis:
+    image: *IMAGE
+    environment:
+      <<: *common-env
+      <<: *fbstest-env
+      CACHE_TYPE: redis
+      CACHE_HOST: redis
+      CACHE_PORT: 6379
+      CACHE_EXPIRE: 600
+    <<: *ports
+
+  fbstest-redis-cluster:
+    image: *IMAGE
+    environment:
+      <<: *common-env
+      <<: *fbstest-env
+      CACHE_TYPE: redis
+      CACHE_HOST: os-opensearch-fbstest-redis-cluster.platform-redis.svc.cloud.dbc.dk
+      CACHE_PORT: 6379
+      CACHE_EXPIRE: 600
+      CACHE_IS_CLUSTER: 1
+    <<: *ports
+
+  cisterne-redis-cluster:
```

Comment:
n00b spm: hvorfor er der ikke en blok for boblebad?

Reply:
Jeg har bare tilføjet de blokke jeg konkret havde brug for. Det er til at køre tests med docker-compose, så jeg har ikke fyldt udfaldsrummet ud ;-).